[SYCLomatic][NFC]Add 'verify the installation' in readme.md (#525)
oneapi-src/SYCLomatic: README.md

ninja install-c2s
```

> Note:For computers with 16G memory, we recommend using 4-6 threads for compilation. Using too many threads may cause out of memory.

Reviewer: ```suggestion
Note: For computers with memory less than 16GB, we recommend using 4-6 threads for compilation. Using too many threads may cause out of memory.
```

> Note: For computers with 16G and below memory, we recommend using 4-6 threads for compilation. Using too many threads may cause out of memory.

Reviewer 2: For computers with 16G or less memory, we recommend using 4-6 threads for compilation. Compiling SYCLomatic with too many threads may cause out of memory.

> Note: For computers with 16G or less memory, we recommend using 4-6 threads for compilation. Compiling SYCLomatic with too many threads may cause out of memory.

Reviewer 3: ```suggestion
Note: For build system with 16G or less memory, recommend to use less than 5 threads for compilation,  you may meet out of memory issue when more than 4 threads used for compilation.
```

```
ninja -j4 install-c2s
```

**build success message**:

> After the compilation is successful, the following information will be output in terminal.

Reviewer: ```suggestion
After the compilation is successful, the following information will be outputted in terminal.
```

> After the compilation is successful, the following information will be outputted in terminal.

Reviewer 2: After a successful build, you should be able to see following message in the terminal output.

```
-- Install configuration: "Release"

> -- Installing: $PATH_TO_C2S_INSTALL_FOLDER/bin/dpct

Reviewer 3: ```suggestion
-- Installing: ... bin/dpct
```

SET CPATH=%PATH_TO_C2S_INSTALL_FOLDER%\include;%CPATH%
```

### Verify the Installation

> Before run SYCLomatic， it is important to verify that dpct/c2s(SYCLomatic binary) can be find in terminal.

Reviewer: ```suggestion
Before running SYCLomatic, it is important to verify that dpct/c2s(SYCLomatic binary) can be found in terminal.
```

> To do this, you need run the following command and get correctly output.

Reviewer: ```suggestion
To do this, you need run the following command and get correct output.
```

> To do this, you need run the following command and get correct output.

Reviewer 2: Before running SYCLomatic, it is important to verify that dpct/c2s(SYCLomatic binary) is installed correctly and PATH is set properly.
To do this, try running the following command and checking if dpct is available after the installation.

```
dpct --version
```

> Note: Ensure the PATH and CPATH are set correctly.

Reviewer 2: If dpct cannot be found, please check the setup of PATH.

> ## Run SYCLomatic

Reviewer 2: ## Run SYCLomatic
### Run dpct command
The help information of dpct is available by running
```
dpct --help
```

### Run c2s command
Get c2s help information by running "c2s --help".

> dpct is an alias command for c2s.

Reviewer 2: c2s is an alias command for dpct.

> ## Test SYCLomatic (Recommended to Contributors)

Reviewer 2: ## Test SYCLomatic

### Run in-tree LIT tests

Note: Certain CUDA header files may need to be accessible to the tool.
After build the SYCLomatic, you can run the list test by: 

ninja check-clang-c2s

> ### Run E2E test suite

Reviewer 2: ### Run E2E test suite (Recommended to Contributors)

> ### Run E2E test suite (Recommended to Contributors)

Reviewer 3: ```suggestion
### Run E2E test suite (Recommend for contributors)
```
